internal/signature/oci.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2022 The Flux authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package signature
```

Comment:
I would rename this module to `oci`.

Reply:
Yeap, I would be inclined to move this code to fluxcd/pkg/oci` but Cosign's huge amount of dependencies scares me, we'll end up with tones of unrelated packages and their CVEs all over the place...

Reply:
> hiddeco: I would rename this module to oci
Yeah, that makes sense to me also, 

> stefanprodan:  I would be inclined to move this code to fluxcd/pkg/oci
Once we have completed the initial implementation, maybe, this one would be the next step, does that make sense?